extensions/ql-vscode/find-deadcode-tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
One problem I had writing this was that our codebase is split into multiple different projects with multiple `tsconfig.json` files that all depend on each other, and all the tools only operate on a single project at a time. Unfortunately just looking at one project you can't tell if a piece of code is unused or not, because it could be used from another project that depends on this one. To work around this I created a new `find-deadcode-tsconfig.json` file that covers the entire repository. I'm not totally sure this means the dead code detection is 100% accurate, and there could be edge cases since the different projects use different rules, but I think it's likely to be good enough.
(Side note: it can't be named `tsconfig.json` because that file already exists in that directory. I could put it in another directory, but I'm not sure which one.)